src/scripts/components/structures/MainHeader.js

```diff
@@ -41,6 +41,8 @@ const HeaderButtons = styled.div`
   justify-content: flex-end;
 `
 
+// foundation/widgets(move?)
```

Comment:
@jrgarou can we delete this?

Reply:
Why remove this @bent0b0x ?

Reply:
Because it is commented out. What is its purpose?